forms-flow-theme/scss/_modal.scss

```diff
@@ -551,3 +506,49 @@
     align-self: stretch;
   }
 }
+
+//reusable multi-select
+.multi-select{
+  .list-group {
+    border: 1px solid var(--ff-base-600);
+    border-radius: var(--radius-0100);
+  
+    .list-group-item {
+      border: none;
+    }
+  }
+  .input-drop {
+    position: absolute;
+    z-index: 100;
+    overflow: auto;
+    border-radius: var(--radius-0100);
+    border-bottom: 1px solid var(--base-base-600);
```

Comment:
pls crosscheck wheteher this is ff-base-600

Reply:
Updated